src/Ball.java

```diff
@@ -22,15 +22,17 @@ class Ball {
     /** The color of the outside of the ball */
     private int borderColor;
 
-    /** Empty constructor to keep the defaults. Only sets up our link. */
+
+
+     // accessors for the radius, diameter, x, and y values 
```

Comment:
This is no longer a true comment, make sure to clean up old unnecessary comments as they become out of date.